client/src/app/components/container/dashboard/presentation/Dashboard.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Row } from 'react-materialize';
+import { Tab, Tabs, TabList, TabPanel } from 'react-tabs';
+import SideNav from '../../../presentation/common/SideNav/index.jsx';
+import DisplayAllBorrowedBooks from '../../booklist/DisplayAllBorrowedBooks.jsx';
+import DisplayAllBooks from '../../booklist/DisplayAllBooks.jsx';
+// import UploadModal from '../../../presentation/common/modal/UploadModal.jsx';
```

Comment:
Zombie code. Try not to push this commented out code. Instead, if you feel you would need it later, keep it in the note app and reference the file name there.